.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,46 @@
+name: "Lint PR title"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            feat
+            fix
+            docs
+            refactor
+            test
+            build
+            ci
+            chore
+            revert
+            BREAKING
+
+          scopes: |
+            api
+            cli
+            orchestrator
+            provider
```

Comment:
Can we also add `ui` and `uibackend` here? To include all service names

Reply:
Will this allow an empty scope?

Reply:
> Will this allow an empty scope?

I have not tested, so I've removed the empty lines to be on the safe side. Although it seems the scopes and types are regex patterns wrapped in `^ $` which will make an empty line a non-matching pattern according to the [docs](https://github.com/amannn/action-semantic-pull-request#configuration).

Reply:
There is a `requireScope: true` in the link you attached! I think that would be useful for example when the type is `docs `

Reply:
Afaik `requireScope: false` is the default.